src/actionHandlers/commands/lockChannel.ts

```diff
@@ -0,0 +1,25 @@
+import { PermissionFlagsBits, TextChannel } from 'discord.js';
+
+import { Command } from './types';
+
+const roles = {
+  member: '1086033687109455989',
+};
+
+export const lockChannel: Command = (interaction) => {
+  const channel = interaction.channel as TextChannel;
```

Comment:
Why didn't u distructure the channel from the interaction as we always do?

Reply:
Because I don't know how to set the type of the channel to TextChannel to access the .permissionOverWrites property